content/support/permanently_deleting_user_data.md

```diff
@@ -0,0 +1,36 @@
+# Permanently deleting user data (GDPR requests, etc.)
+
+Sometimes users will ask us to remove their account data. This can be just because they want it done, or as a GDPR compliance request, etc.
+
+### Submitting a request
+
+Anyone may request to have their user data permanently deleted from Sourcegraph.com, whether for a GDPR request or otherwise. We treat all requests the same way and with urgency.
+
+You can request this via support@sourcegraph.com
+
+### Support: Processing a data deletion request
+
+The support team member is responsible for delegating to other teams. If you're not the support team, you should inform the user that only our support team support@sourcegraph.com can handle such requests.
+
+[Create an issue in the sourcegraph/customer repository](https://github.com/sourcegraph/customer/issues/new/choose) and follow the "**Customer support: Delete user data**" checklist issue template.
```

Comment:
I think I am missing where I can review this template to see what all is required of CS. Can you help me understand where the full list is?